CONTRIBUTING.md

```diff
@@ -48,9 +48,6 @@ The whole project is covered by a set of static analysis tools, linters and test
 # launches the detekt static analysis for all modules
 ./gradlew detektAll
 
-# launches the ktlint format check for all modules
```

Comment:
Can we mention the ktlint cli here instead to make sure contributors know that this is a check they should use: 

```console
ktlint -F "**/*.kt" "**/*.kts" '!**/build/generated/**' '!**/build/kspCaches/**'
```